spark/v3.2/spark/src/jmh/java/org/apache/iceberg/spark/data/parquet/SparkParquetWritersFlatDataBenchmark.java

```diff
@@ -85,7 +86,7 @@ public void setupBenchmark() throws IOException {
     dataFile.delete();
   }
 
-  @TearDown
+  @TearDown(Level.Invocation)
```

Comment:
not sure but I think both should be Iteration rather than Invocation

Reply:
Iteration and Invocation should have the same effect since BenchmarkMode here is SingleShotTime, but I think is should be Invocation since each Invocation will produce a file.